packages/tracing/src/browser/web-vitals/getLCP.ts

```diff
@@ -46,30 +46,37 @@ export const getLCP = (onReport: ReportHandler, reportAllChanges = false): void
 
     // If the page was hidden prior to paint time of the entry,
     // ignore it and mark the metric as final, otherwise add the entry.
-    if (value < firstHidden.timeStamp) {
+    if (value < visibilityWatcher.firstHiddenTime) {
       metric.value = value;
       metric.entries.push(entry);
-    } else {
-      metric.isFinal = true;
     }
 
-    report();
+    if (report) {
```

Comment:
Is this ever called? I only see it being typed above `let report: ReturnType<typeof bindReporter>;

Reply:
It's set at a few lines below.